flang/lib/Optimizer/CodeGen/CodeGen.cpp

```diff
@@ -3559,6 +3560,14 @@ class FIRToLLVMLowering
     // as passes here.
     mlir::OpPassManager mathConvertionPM("builtin.module");
 
+    bool isAMDGCN = fir::getTargetTriple(mod).isAMDGCN();
+    // If compiling for AMD target some math operations must be lowered to ocml
```

Comment:
Nit: would it be possible to get the full name for ocml in the comments? Maybe a weird request, just unfamiliar with the library so would be handy to have in the comment! if not that's not a big deal.

Reply:
I think ocml stands for OpenCL Math Library, so I'm not sure that will help or confuse more. I should perhaps use "AMD GPU math library" or something like that.